src/test/ui/pub/pub-ident-fn-with-lifetime.rs

```diff
@@ -0,0 +1,6 @@
+pub   foo<'a>(_s: &'a usize) -> bool { true }
+//~^ ERROR missing `fn` for method definition
```

Comment:
Outside an `impl` it cannot possibly be a method so you should imo either 1) say "function definition", 2) check if there's a receiver `(&mut?)? self` and then say "method definition" or otherwise say "function definition.

Reply:
Done